src/client/components/pages/entities/edition-table.js

```diff
@@ -145,11 +167,18 @@ function EditionTable({editions, entity, showAdd}) {
 EditionTable.displayName = 'EditionTable';
 EditionTable.propTypes = {
 	editions: PropTypes.array.isRequired,
-	entity: PropTypes.object.isRequired,
-	showAdd: PropTypes.bool
+	entity: PropTypes.object,
+	selectedEntities: PropTypes.array,
+	showAdd: PropTypes.bool,
+	showCheckboxes: PropTypes.bool,
+	toggleRow: PropTypes.func
```

Comment:
```suggestion
	onToggleRow: PropTypes.func
```

This should be changed wherever that function is passed to a child component, to clearly identify that it's a event  function